core/editor/templates/editor-body-v1.view.php

```diff
@@ -10,6 +10,8 @@
 $notice = Plugin::$instance->editor->notice_bar->get_notice();
 ?>
 
+<h1 class="elementor-screen-only"><?php echo esc_html__( 'Elementor Editor', 'elementor' ); ?></h1>
```

Comment:
Please change the title to: `Edit <$post_title> - Elementor`
Use `$post_id` as a fallback if there is no title.

Reply:
No problem, I will add the post title, but I think **Edit "%s" with Elementor** title makes much more sense.